packages/sdk/src/lib/dataProtectorSharing/subgraph/getRentalsQuery.ts

```diff
@@ -0,0 +1,45 @@
+import { gql } from 'graphql-request';
+import { throwIfMissing } from '../../../utils/validators.js';
+import type { GetRentalsGraphQLResponse } from '../../types/graphQLTypes.js';
+import type { GetRentalsParams } from '../../types/index.js';
+import type { SubgraphConsumer } from '../../types/internalTypes.js';
+
+export const getRentalsQuery = async ({
+  graphQLClient = throwIfMissing(),
+  renterAddress,
+  protectedDataAddress,
+  includePastRentals = false,
+}: SubgraphConsumer & GetRentalsParams): Promise<GetRentalsGraphQLResponse> => {
+  const rentalsQuery = gql`
+    query ($where: Rental_filter) {
```

Comment:
where does Rental_filter type come from ?

Reply:
We can see it in the schema docs for example:

<img width="584" alt="Capture d’écran 2024-03-29 à 10 36 34" src="https://github.com/iExecBlockchainComputing/dataprotector-sdk/assets/4280765/8a130d04-aef5-491f-bc8a-138756303855">

Reply:
We can see it in the graphiql schema docs for example:

<img width="633" alt="Capture d’écran 2024-03-29 à 10 38 45" src="https://github.com/iExecBlockchainComputing/dataprotector-sdk/assets/4280765/deed4b07-c6fe-49fb-ab9e-0085eca426fe">